Added release highlights to ESQL

craigtaverner opened the pull request:
The original PR was labeled `release highlight` but did not yet have the highlight text. This PR adds a first draft of that text, extracted from the ESQL blog.

craigtaverner committed:
Added release highlights to ESQL

The original PR was labeled `release highlight` but did not yet have the highlight text.
This PR adds a first draft of that text, extracted from the ESQL blog.

elasticsearchmachine commented:
Pinging @elastic/es-docs (Team:Docs)

elasticsearchmachine commented:
Pinging @elastic/es-ql (Team:QL)

elasticsearchmachine commented:
Pinging @elastic/elasticsearch-esql (:Query Languages/ES|QL)

craigtaverner commented on docs/changelog/98309.yaml:
```diff
@@ -3,3 +3,31 @@ summary: "Integrate Elasticsearch Query Language, ES|QL"
 area: Query Languages
```
Should we change this to ES|QL to match all other ES|QL PR's (right now this is the only ES|QL PR in `Query Languages`)?

alex-spies reviewed (approved):
Heya, I just proof-read this for good measure; LGTM. Since this is already coming from a blog post that was certainly iterated on, I didn't really find anything except one small remark.

craigtaverner commented on docs/changelog/98309.yaml:
```diff
@@ -1,5 +1,33 @@
+      and use queries to create custom alerts.
+
+    [role="screenshot"]
+    image:images/esql/illustration_esql_kibana_wireframe.png[ES|QL usage in Kibana Discover]
```
Agreed!

elasticsearchmachine commented:
Hi @craigtaverner, I've created a changelog YAML for you. *Note* that since this PR is labelled `release highlight`, you need to update the changelog YAML to fill out the extended information sections.

craigtaverner committed:
Add new screenshot for kibana ESQL

craigtaverner committed:
Remove ESQL prefix from all ESQL changelog entries

Unlike the original PR's these entries are sorted under an `ES|QL` subsection, so this is redundant, and visually jarring.
